File: Hafta1/mario.c
#include <cs50.h>
#include <stdio.h>

int main(void)
{

int h=0;

do
{

h=get_int("Enter the height.\n");

}while(h<1 || h>8);

for(int i=1;i<=h;i++)
{
for(int j=h;j>i;j--)
{
    printf(" ");
}

for(int k=1;k<=i;k++)
{
    printf("#");

}

printf("\n");
}
return 0;
}

File: Hafta6/mario.py
from cs50 import get_int

key=0

while(key==0):
    h=get_int("Height:")
    if(h>0 and h<9):
        for i in range(1,h+1):
                
            for j in range(h - i):
                print(" ", end = "")
            for k in range(i):
                print("#", end = "")
            print("")
            key=1
        
    else:
        key=0
    
        
    


    

        

